Updates/841_Saltgurka-Honor_Hold_Updates.sql

```diff
@@ -0,0 +1,1092 @@
+-- ----------------------------------------------------------
+-- Model and equipment updates
+-- ----------------------------------------------------------
+-- Update EquipmentTemplate for all Honor Hold Defenders. Entry 20513 (the two guards next to the destroyed keep) had the correct ones, all others were wrong for some reason.
+UPDATE `creature_template` SET `EquipmentTemplateId`=1195 WHERE `entry`=16842;
+
+-- Also the human model was missing from their creature_template
+UPDATE `creature_template` SET `ModelId1`=16387,`ModelId2`=16388,`ModelId3`=16389,`ModelId4`=16390 WHERE `entry` IN (16842,20513); -- 16390

```

Comment:
So basically how it was before was that the two guards next to the destroyed keep were on their separate template and they only had the human models (16387 and 16388). All other guards only had the dwarf models (16389 and 16390). 
All I did was combining them on both templates so they can be dwarf female, dwarf male, human male, human female. 

The reason I removed the gender link is because Watch Commander Leonus uses 16387 and he should never be female. There's one more quest giver outside of Honor Hold with the same issue. Just removing the gender link was the easiest solution imo